src/components/GameContainerbasic.js

```diff
@@ -0,0 +1,193 @@
+//game container basic version
+
+import React, {Component} from 'react';
+import styled from 'styled-components';
+import Server from '../server.js';
+const SERVER = 'http://localhost:4000';
+import io from 'socket.io-client';
+const socket = io(SERVER);
+
+
+const BetInput = styled.input`
+  display: block;
+`;
+
+const CardInput = styled.input`
+  display: block;
+`;
+
+
+//displays messages
+function Messages(props){
+  return(
+    <div id="Messages">
+      <p>{props.messages}</p>
+    </div>
+  )
+}
+
+function Hand(props){
+  return(
+    <div id="Hand">
+      <p>{props.hand}</p>
+    </div>
+  )
+}
+
+class GameContainerbasic extends Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      gameState: {joined: false},
+      messages: null,
+      id: props.match.params.id,
+    }
+    this.loadGame();
+    //this.GameState();
+    this.socket = socket.connect(()=>{
+      console.log("i am connected");
+    });
+
+    this.socket.on("update", ()=>{
+      console.log('client got update');
+      this.loadGame();
```

Comment:
We should accept a JSON string here that will look just like a response from a normal http request and we will update state accordingly. `loadGame()` makes another unnecessary http request when we can just get it from whatever is emitted in the update.